### DISCUSSION TITLE
Synth dialog!!

### DISCUSSION THREAD

**Discussion by spamatica** [muse-sequencer/muse#1194] [Team Posts]
Oh man, it makes me both happy and sad with all the improvements. Happy with all the progress, all new features and fixes, but a bit sad that I can't seem to help more... 
Maybe I can help make another pre-release anyway, we should probably do one.

The new dialog is a great addition! It will be my new way of instantiating synths (especially the search field, makes all the difference for me).
What do you think about opening this dialog when double clicking on the track list background? ...trying to find ways to get an even faster workflow (and a shortcut, why not).
And would it be possible to add the other kinds of track types to the searchable list?

It would be truly awesome if it was possible to achieve a workflow like:
1. Ctrl+Shift+A (or whatever is available) to open the dialog
2. type aux in the search field (which then would highlight the aux track type)
3. press enter to instantiate it
That would be close to nirvana :)

In other news there's a snowstorm here at the moment, biggest we had in years. Earlier today I took a trip to work for a few hours, immediately regretted it. Took my nearly an hour of showelling show to get my car from the driveway, and almost feared for my life out in traffic, cars stuck on side roads all over the place.

**Comment by kybos** [comment]
> What do you think about opening this dialog when double clicking on the track list background? ...trying to find ways to get an even faster workflow (and a shortcut, why not).
> And would it be possible to add the other kinds of track types to the searchable list?

It's a synth selection so only relevant for the synth track type. A track selection is quite a different thing. 
But I agree that it should be possible to add tracks fast and without using the mouse, as it's a very frequent action.
You know that there are shortcuts for adding midi and wave tracks already? Also drum is there but not assigned out-of-the-box. So it would be good to add the other track types, find shortcuts for them, and also (very important) show the shortcuts in the track menu so the users know that they exist.

Yes, even here in this rather warm area of Germany it was snowing quite frequently in the last days, also today. But far from what you describe, of course ;-). Glad you got home without accident.

**Comment by spamatica** [comment]
Den tis 12 jan. 2021 kl 20:12 skrev kybos <notifications@github.com>:

> What do you think about opening this dialog when double clicking on the
> track list background? ...trying to find ways to get an even faster
> workflow (and a shortcut, why not).
> And would it be possible to add the other kinds of track types to the
> searchable list?
>
> It's a synth selection so only relevant for the synth track type. A track
> selection is quite a different thing.
>
Yeah, I was thinking of a slight change from "Add Synth..." to "Add...", I
don't think it requires so many changes in the UI, but if it is complicated
under the hood that is a different thing.

> But I agree that it should be possible to add tracks fast and without
> using the mouse, as it's a very frequent action.
> You know that there are shortcuts for adding midi and wave tracks already?
> Also drum is there but not assigned out-of-the-box. So it would be good to
> add the other track types, find shortcuts for them, and also (very
> important) show the shortcuts in the track menu so the users know that they
> exist.
>
Indeed I know the shortcuts and use several of them. Especially
Ctrl+Shift+D to duplicate a track. When recording audio tracks I usually
record lots that are connected to the same input and the same group track
and this takes away most of the tedious reconnecting.

> Yes, even here in this rather warm area of Germany it was snowing quite
> frequently in the last days, also today. But far from what you describe, of
> course ;-). Glad you got home without accident.
>
Ah, yes, I saw the videos from Spain, all over europe I guess. One
difference is that we are used to the snow at least, but this was more than
I bargained for :D Still haven't cleared the driveway.

**Comment by terminator356** [comment]
Here in the land of beaver moose snow and hockey we have the opposite.
Zero snow on the ground at Christmas, not much now. Pretty dry here, warmer than normal temps.

Today it was announced we are going into another COVID state of emergency.
We are back at square one like when it all started, except even stricter.
It amounts to essentially the same as an 8 PM curfew, something I've never experienced before.
They say the 'numbers' are going waaay in the wrong direction. They must now take immediate action.
The province of Quebec (next door to us in Ontario) is already under strict curfew.
Everyone is stressed out.

**Comment by spamatica** [comment]
Oh man, hang in there. At least you can talk to us in our little bubble.
It's stricter here too but it's never been that strict here in sweden... opinions about that of course vary.
From yesterday the snow has completely overshadowed the covid crisis. I don't think I have ever experienced so much snow in such a short time.

**Comment by kybos** [comment]
We've had curfew at 8 PM for weeks now, and many other similar measures. But worst of all is the home-schooling... ;-).

**Comment by kybos** [comment]
Back to the topic:

It's recommendable to reset the keyboard shortcuts to the default to get the complete new functionality (there is now a button for exactly that in the Keyboard shortcuts dialog, I added it some time ago). Otherwise the new shortcuts are overridden by the old saved (partly empty) ones.
I think now creating all types of new tracks is as easy and fast as it can be, and the shortcuts are also very easy to memorize.

**Comment by spamatica** [comment]
Alright, super cool, will check it out

Den fre 15 jan. 2021 kl 17:59 skrev kybos <notifications@github.com>:
